deploy/role_binding.yaml

```diff
@@ -9,3 +9,15 @@ roleRef:
   kind: Role
   name: smart-gateway-operator
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: smart-gateway-operator
+subjects:
+- kind: ServiceAccount
+  name: smart-gateway-operator
+roleRef:
```

Comment:
I'm slightly surprised that the above works without a namespace field for the ServiceAccount. I assume it's putting the current namespace as the default here?

Reply:
Yea I suspect this works because of the way things are built via OLM. It's been like this for a while, so I suspect it works :) Wouldn't work if you applied this directly though likely.

Reply:
If you apply this by hand, it complains that a namespace is missing.